src/patterns/__snapshots__/ec2-app.test.ts.snap

```diff
@@ -829,10 +829,6 @@ Object {
           },
         ],
         "Tags": Array [
-          Object {
```

Comment:
The app tag is only being removed from `WazuhSecurityGroup`s. This construct is [implemented as a singleton anyway](https://github.com/guardian/cdk/blob/a12758512cc6942cfe8234a8ef413edb04596e0c/src/constructs/ec2/security-groups/wazuh.ts#L9-L11) (i.e. it might be shared by multiple apps in the same stack) so it doesn't make sense to add the app tag here in all cases anyway!